providers/openebs/installers/operator/0.6/ansible/openebs_cleanup.yaml

```diff
@@ -0,0 +1,44 @@
+# TODO 
+# Delete openebs operator
+
+- hosts: localhost
+  connection: local
+
+  vars_files:
+  - vars.yaml
+
+  tasks:
+    - block:
+          - name: Cleaning openebs operator
+            shell: "{{ kubeapply }} delete -f {{ openebs_operator_link }}" 
+            args:
+              executable: /bin/bash
+            ignore_errors: True
+
+          - name: Confirm pods has been deleted
+            shell: kubectl get pods --all-namespaces
+            args:
+              executable: /bin/bash
+            register: result
+            until: "item not in result.stdout"
+            with_items:
+              - "maya-apiserver"
+              - "openebs-provisioner"
+              - "openebs-snapshot-operator"
+            delay: 30
+            retries: 100
+
+          - name: Confirm that namspace has been deleted
+            shell: kubectl get namespaces
+            args:
+              executable: /bin/bash
+            register: result
+            until: "namespace not in result.stdout"

```

Comment:
Just added a minor commit here to remove the single quotes on namespace - as it would evaluate this to a string called namespace instead of the var specified.